vyper/compiler.py

```diff
@@ -139,6 +141,6 @@ def compile_codes(codes, output_formats=['bytecode'], output_type='list', exc_ha
         raise Exception('Unknown output_type')
 
 
-def compile_code(code, output_formats=['bytecode']):
+def compile_code(code, output_formats=None):
```

Comment:
@pipermerriam does it feel wrong to you to just use `output_formats=('bytecode',)` here, in other words use an immutable data structure to mitigate the risk?

Reply:
Yes, I chose this because of the other function also specifying this as a default and thus, by using `None` I could avoid duplicating the default parameter across the two functions.

I could get around this by encoding it as a constant, but this felt better to me.